.travis.yml

```diff
@@ -76,6 +77,12 @@ install:
         ;;
     esac
 
+before_script:
+  # fail the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
```

Comment:
Can you add a comment here that false positives can be suppressed with `# noqa` or preferably `# noqa: <errorcode>`?